script/webots-firejail

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Run webots controllers in jails
+export WEBOTS_FIREJAIL_CONTROLLERS=1
+
+# Used by `runtime.ini` to pass through all relevant files into jail
+export SR_SIM_JAIL_PATHS=$(realpath -e ${PWD}/$(dirname $0)/../../)
```

Comment:
I'm pretty sure that use of `$PWD` here is redundant (and possibly incorrect?).

Would it be worth having some interim variables so it's a bit clearer what this does?

```suggestion
REPO_ROOT=$(dirname $0)/..

# We want the Firejailed process to have access to the siblings of the
# simulation directory because the  competitors code lives in sibling directories.
REPO_PARENT=$REPO_ROOT/..

# Used by `runtime.ini` to pass through all relevant files into jail
export SR_SIM_JAIL_PATHS=$(realpath -e $REPO_PARENT)
```